Lib/test/test_named_expression.py

```diff
@@ -166,7 +166,7 @@ def test_named_expression_invalid_rebinding_comprehension_inner_loop(self):
                 with self.assertRaisesRegex(SyntaxError, msg):
                     exec(f"lambda: {code}", {}) # Function scope
 
-    @unittest.expectedFailure # TODO RustPython
+    @unittest.expectedFailure # TODO RustPythonskip
```

Comment:
is this a mistake?

Reply:
That was not intended. I fix it when completing the stuff.

Reply:
Let me know when you complete things. Keeping it as unresolved will be helpful to not miss it before merging.